management-app/src/main/java/com/sdevelopment/skt/management/controller/ProductController.java

```diff
@@ -0,0 +1,56 @@
+package com.sdevelopment.skt.management.controller;
+
+import com.sdevelopment.skt.common.domain.Product;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.ModelMap;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import javax.validation.Valid;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+@Controller
+public class ProductController {
+
+    @RequestMapping("/create-product")
+    public String createProduct(ModelMap model) {
+        Product product = new Product();
+        model.addAttribute("product", product);
+
+        return "newProduct";
+    }
+
+    @RequestMapping("/products")
+    public String listProducts(ModelMap model) {
+        List<Product> products = new LinkedList<>();
+
+        Product p1 = new Product();
+        p1.setName("Product 1");
+
+        Product p2 = new Product();
+        p1.setName("Product 2");
+
+        products.add(p1);
+        products.add(p2); // Two elements just to show in the view.
+
+        model.addAttribute("products", products);
+
+        return "listProducts";
+    }
+
+    @RequestMapping(value = "/product-form", method = RequestMethod.POST)
+    public String processNewProduct(@Valid @ModelAttribute("product")Product product,
+                                    BindingResult result, ModelMap model, RedirectAttributes redirectAttributes) {
+
```

Comment:
I think this method is not doing anything

Reply:
Not yet. From there I'm going to send the product to the Queue.

Reply:
I will.

Reply:
Already implemented. Here I save the product which means send it to the queue.